.changeset/pretty-trains-appear.md

```diff
@@ -0,0 +1,26 @@
+---
+'example-app': patch
```

Comment:
Looks like there's some packages in here that don't require a changeset.

Can remove `example-app` `example-backend` `e2e-test` and `storybook`

Reply:
Addressed in https://github.com/backstage/backstage/commit/97b8c8c8fc6a44bc19ad9850ae0ef8890f66c890